pegasus/sites.v3/code.org/public/js/resources-tabs.js

```diff
@@ -0,0 +1,25 @@
+// Works with the "resources_tabs.haml" partial.
+// Styles that show and hide sections are in "design-system-pegasus.scss"
+// under the "div.resources-tabs" selector.
+
+const tabs = document.querySelectorAll(".tab");
```

Comment:
Will this be the only spot tabs are used on this page?

Reply:
For now yes, but do you think I should call this more specifically like  `document.querySelectorAll(".resources-tabs button.tab")`?

Reply:
I think being more specific could help avoid future issues as this page gets developed! Just in case tabs are used elsewhere

Reply:
Cool, ty! I'll update it now 👍